src/documents/tutorial-userInterfaceOptions/UserInterfaceOptions.md

```diff
@@ -16,44 +16,71 @@ UI Options does three things:
 
 ![Screen shot of the UI Options Component](../images/uio-showcase.png "Screen shot of the UI Options Component")
 
-This tutorial assumes that:
-
-* you are already familiar with HTML, Javascript and CSS,
-* you are familiar with what the UI Options preferences editor is and does, and
-* now you just want to know how to add it to your website.
+You can try the [UI Options demo here](http://build.fluidproject.org/infusion/demos/uiOptions/).
 
 ## Scenario
 
 You're putting together a website that you know will have a diverse audience. You'd like to allow your visitors to
 customize the presentation of the site to their individual needs, by enlarging the text or increasing the visual
-contrast, for example. This tutorial will show you how to add the Infusion [UI
-Options](http://build.fluidproject.org/infusion/demos/uiOptions/) component to your site.
+contrast, for example. This tutorial will show you how to add the Infusion UI Options component to your site.
 
 These are the basic steps to add UI Options to your website:
 
-1. [Download and install the Infusion library](#download-and-install-the-ui-options-library)
+1. [Download and install the UI Options library](#download-and-install-the-ui-options-library)
 2. [Prepare your page](#prepare-your-page)
     * [The Sliding Panel](#the-sliding-panel)
     * [The Table of Contents](#the-table-of-contents)
 3. [Add dependencies to the page](#add-dependencies-to-the-page)
 4. [Add the UI Options component](#add-the-ui-options-component)
 
-The rest of this tutorial will explain each of these steps in detail.
+The rest of this tutorial will explain each of these steps in detail. This tutorial also assumes that you can work with HTML, Javascript and CSS.
 
 ## Download and install the UI Options library
 
+Get Infusion using one of these methods:
+* [Zip Package](https://github.com/fluid-project/infusion/releases/download/v2.0.0/infusion-uiOptions-2.0.0-source.zip)
```

Comment:
Rather than pointing at a specific zip file, what do you think about pointing at the releases page on GitHub https://github.com/fluid-project/infusion/releases Then you can just mention to pick a zip package from one of the releases so we don't have to update this tutorial for each Infusion release if nothing else changed.